config.yaml

```diff
@@ -0,0 +1,7 @@
+server:
+  host: "localhost"
+  port: 8080
+
+database:
```

Comment:
Please include (minimal) documentation on the expected database setup.

Reply:
Actually since there is no database yet, it makes no sense to include this here yet